### ISSUE TITLE
Info: how to get a specfic lap of a pilot

### ISSUE THREAD

**Issue by Xwang1976** [Ax6/Fast-F1#9]
How can I get a specific lap for a pilot and not only the fastest one?

**Comment by Ax6** [comment]
```python
import fastf1 as ff1

quali = ff1.get_session(2019, 'Mexico', 'Q')
laps = quali.load_laps()
lec_laps = laps.pick_driver('LEC')
print(lec_laps)
```

lec_laps contains all laps from leclerc. It is a `Laps` object which can be treated just like a pandas DataFrame.

- For other `pick` operations and what is the `Laps` class look here https://ax6.github.io/Fast-F1/core.html#fastf1.core.Laps
- If you have troubles with dataframes, have a look here https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.html.

**Comment by Xwang1976** [comment]
No, I want to be able to determine the number of laps and then get all the laps (inside a for loop) to be able to analyse all the telemetries (an not only the fastest lap).
I will study the two links above and I hope I'll solve the issue.

**Comment by Ax6** [comment]
Ok, assuming you don't care about slow laps (usually in-laps and out-laps) you could do something like this:

```python
import fastf1 as ff1

quali = ff1.get_session(2019, 'Mexico', 'Q')
laps = quali.load_laps()
lec_laps = laps.pick_driver('LEC').pick_quicklaps()

for idx in lec_laps.index:
    lap = lec_laps.loc[idx]
    print(lap)
    """ Anything you else want to see about the lap.
    To start, I'd suggest to debug and stop to this line or use
    a `breakpoint()` so you can play with the lap object"""
```
